metrics/tag.go

```diff
@@ -76,6 +76,16 @@ func MustNewTag(k, v string) Tag {
 	return t
 }
 
+// MustNewTagWithFallbackValue returns the result of calling NewTag, but panics only if the key input is invalid.
+// If the value input is invalid, it will be replaced with the fallback value.
+func MustNewTagWithFallbackValue(k, v, fallback string) Tag {
```

Comment:
- I would prefer a name like NewTagWithFallbackValue since Must* functions usually panic (e.g. the standard library's regexp.MustCompile)
- Let's mention in the comment that the fallback is passed through MustNewTag and will panic if invalid.